docs/api/rpc-experimental.md

```diff
@@ -7,81 +7,18 @@ sidebar_label: Experimental Methods
 <blockquote class="warning">
 <strong>heads up</strong><br><br>
 
-Experimental API methods are feature-complete, but we do not guarantee them to be backward-compatible or exist forever. For example, we are going to remove `EXPERIMENTAL_genesis_records` API soon and `EXPERIMENTAL_genesis_config` does not have a stable structure (we used to change it quite frequently); we are committed to never do such changes for the non-EXPERIMENTAL endpoints. 
+Experimental API methods are feature-complete, but we do not guarantee them to be backward-compatible or exist forever. 
 
-EXPERIMENTAL endpoints are useful for API evaluation on the real network. While the API is in EXPERIMENTAL state we collect the feedback and can iterate on the input/output structure.
+These endpoints are useful for API evaluation on the real network. While the API is in EXPERIMENTAL state, we collect feedback and can iterate on the input/output structure.
 
 </blockquote>
 
-## Changes in Block
```

Comment:
Do we deprecate rpc-experimental? I see some methods being moved to rpc.md while others are left intact.

Reply:
Yes, @bowenwang1996 asked me to consolidate to one document. The ones left in `rpc-experimental.md` still need example snippets created but was told I can hold off on that for now.

Reply:
I see, I just found it confusing that `EXPERIMENTAL_changes_in_block` documentation has been moved while `EXPERIMENTAL_changes` left behind :thinking:

Reply:
Totally understand, it will be my goal to get this moved over in the very near future :)